devices/GnssDevice/Nmea0183Parser.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Common.GnssDevice
+{
+    /// <summary>
+    /// Provides methods for parsing NMEA0183 data from a Gnss device.
+    /// </summary>
+    public static class Nmea0183Parser
+    {
+        /// <summary>
+        /// Gets or sets a list of NMEA0183 data objects.
+        /// </summary>
+        public static INmeaData[] MneaDatas { get; set; }
+
+        /// <summary>
+        /// PArses a string and return the parsed NmeaData object.
+        /// </summary>
+        /// <param name="inputData">A valid MNEA string.</param>
+        /// <returns>Parsed NmeaData object if any or null.</returns>
+        public static INmeaData Parse(string inputData)
+        {
+            var data = inputData.Split(',');
+            foreach (INmeaData mnea in MneaDatas)
+            {
+                if (mnea.Name == data[0])
+                {
+                    return mnea.Parse(inputData);
+                }
+            }
+
+            return null;
+        }
+
+        /// <summary>
+        /// Parses the GNGLL NMEA0183 data from a Gnss device.
+        /// </summary>
+        /// <param name="inputData">The valid GNGLL data string starting with $GNGLL.</param>
+        /// <returns>The parsed GngllData object.</returns>
+        public static GngllData ParaseGngll(string inputData)
```

Comment:
@Ellerbach Should we remove it from parser?

Reply:
yes, yes, again, sorry, miss to push a commit yesterday, please check now, should be all good.